.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER_PATH='http:localhost:8080/'
```

Comment:
I am not sure, but (sigle)quotes are not required here.

Reply:
Just make sure it works with this annotation.

Reply:
works 👍